Performance/.gitignore

```diff
@@ -0,0 +1,4 @@
+Generated/
+Results/
+Sources/*+Generated.swift
```

Comment:
These files are generated in /Performance/Generated, so what goes on in here shouldn't have any effect on general building where it only looks in /Sources.

And actually that line 3 in the .gitignore can be removed, it's a remnant of another attempt that didn't work where I tried to build the performance tester with Swift PM as well.